io.gdevelop.ide.yml

```diff
@@ -1,12 +1,14 @@
 # This is the Flatpak manifest for GDevelop.
 app-id: io.gdevelop.ide
 base: org.electronjs.Electron2.BaseApp
-base-version: "22.08"
+base-version: '22.08'
```

Comment:
23.08 ????

Reply:
Orthogonal to this change; 22.08 has another 8-9 months' life in it.

Reply:
I pushed the patch to increment this number, since it seems to work. (Though I maintain that it's orthogonal to both this and the ARM64 change.)